configs/platforms/fedora-26-x86_64.rb

```diff
@@ -0,0 +1,11 @@
+platform "fedora-26-x86_64" do |plat|
+  plat.servicedir "/usr/lib/systemd/system"
```

Comment:
Couldn't you just do something like
```
plat.instance_variable_set(:@name, 'fedora-26-x86_64')
```
in the old platform.rb files?

Reply:
Does that allow us to make the platform name depend on the project being built? Agent projects older than master still need the `f`.

Reply:
I think I like this approach better, but I'm curious if you have reasons for your suggestion @ekinanp

Reply:
I think so, you should be able to do that in the project files, something like:
```
if plat.name =~ /fedora-(\d+)/
  plat.instance_variable_set(:@name, fedora-f\1-x86_64) 
end
```
in e.g. agent-runtime-1.10.x, agent-runtime-5.3.x.

@mwaggett Just seems overkill adding new platform files for something like that is all. I don't have a strong opinion on either approach though.

Reply:
In my opinion, this seems easier and cleaner than your conditional. I'd love for all the `f` conditionals to be gone asap. When we EOL puppet5 (which I know will be a while), it seems easiest if we could just EOL the f-containing platform configs like any other platform.

Reply:
Yeah, when you think about it as a separate platform it makes sense. Then my only other suggestion is adding comments in the plat. files indicating why we have things set-up the way we do (e.g. in the'f' prefix ones, something like "We need this for older agent projects, we're planning on removing it once we EOL puppet5" would be useful).

Reply:
This approach seems to work in cursory local tests, but I want to make sure they produce expected results everywhere. I think I'm going to build and ship packages for the updated platforms ad-hoc, so that other projects can consume them now, and wait for PR tests to finish (then build pdk and agents) for these changes.

Reply:
@mwaggett  Seeing the changes that Casey just pushed up, I think putting it in the project files is better (if it works). The main reason is b/c fedora-f27 and fedora-27 are not meant to be different platforms, they are the same platform - fedora-27. The reason we need the 'f' prefix in the former is because older agent projects require it. That's a project-induced restriction. Hence, consolidating the 'f' prefix should belong in the project files.